app/controllers/application_pets_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationPetsController < ApplicationController
+
+  def index 
+    pet = Pet.find(params[:pet_id])
+    @applications = pet.applications
```

Comment:
Makes is easy to use applications as an instance variable instead of calling it on a pets instance variable